.github/pull_request_template.md

```diff
@@ -1,7 +1,9 @@
 Pull request checklist:
- - [ ] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
- - [ ] All text is internationalized
- - [ ] There are no linter errors
- - [ ] New features support all states (loading, error, etc) 
+
+- [ ] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
+- [ ] All text is internationalized
+- [ ] There are no linter errors
+- [ ] New features support all states (loading, error, etc)
+- [ ] Which JIRA ticket(s) and/or github issues does this PR address?
```

Comment:
GitHub

Reply:
Updated in https://github.com/WildMeOrg/codex-frontend/pull/421/commits/e1482a0d6862959bd1a9041d4443af8b1a2cc007